ssl/test/test_config.cc

```diff
@@ -306,8 +306,9 @@ std::vector<Flag> SortedFlags() {
       BoolFlag("-renegotiate-explicit", &TestConfig::renegotiate_explicit),
       BoolFlag("-forbid-renegotiation-after-handshake",
                &TestConfig::forbid_renegotiation_after_handshake),
-      BoolFlag("-enable-all-curves",
-               &TestConfig::enable_all_curves),
+      // TODO(guywald): Bring it back.
```

Comment:
Is that something just for your own testing?

Reply:
Yes :) Intend to circle back to this, should have addressed it in the PR description.
Just wanted to get it working at first. In high likelihood it affects the tests as well.

Reply:
Yes, the `enable_all_curves` flag was removed during the conflict, just wanted to get it working at first.
Should have addressed it in the PR description.
Introduced it back, thank.